rados/ioctx.go

```diff
@@ -342,6 +343,7 @@ func (ioctx *IOContext) ListObjects(listFn ObjectListFunc) error {
 			item := results[i]
 			listFn(C.GoStringN(item.oid, (C.int)(item.oid_length)))
 		}
+		C.rados_object_list_free(C.ulong(ret), res)
```

Comment:
Wouldn't this be better as:
```suggestion
		C.rados_object_list_free(C.size_t(ret), res)
```
In order to best match the type signature of [rados_object_list_free](https://github.com/ceph/ceph/blob/bd8b7aa4dabda2a3de36f98406d691755d669882/src/include/rados/librados.h#L1248) ?

Reply:
Good point, updated :+1: